DataDog/libdatadog
# RFC: Allow verify trace-protobuf to continue on error

## What does this PR do?

This PR changes the "Verify trace-protobuf" github action to include "continue-on-error", so that unrelated PRs don't get marked as having failed CI when this action triggers.

Additionally, it makes use of the
`mainmatter/continue-on-error-comment@v1` action to turn these errors into a comment that gets added into the PR, so that we don't miss that this action failed -- we just make failures less noisy.

## Motivation:

I like having a green CI, and it's annoying to have a red CI because this action failed, since for many libdatadog PRs we don't care (e.g. this only impacts the serverless agent).

Thus, I decided to experiment with this alternative approach. As the PR title says -- comments welcome! Do you think this is reasonable?

## Additional Notes:

N/A

## How to test the change?

I'm hoping the action will start triggering for this PR itself, so we can see it running.

## For Reviewers
